pkgs/tools/wayland/wl-gammactl/dont-need-wlroots.diff

```diff
@@ -0,0 +1,20 @@
+diff --git a/meson.build b/meson.build
+index 5253f52..f5ff82e 100644
+--- a/meson.build
++++ b/meson.build
+@@ -6,7 +6,6 @@ project(
```

Comment:
shoudlnt this be applied upstream if they dont use it ?

Reply:
Yes, I open a pr to upstream https://github.com/mischw/wl-gammactl/pull/15

But the upstream is no longer actively maintained...

Reply:
are you fixing  the package to get rid of an old wlroots derivation or do you actually use the package ? just curious if there is some active fork, anyway last commit is from 2021 so i suppose it's recent enough.

Reply:
I use this package to test my server side implement https://github.com/vioken/waylib/pull/220， maybe I will write a qml application in the future as an alternative